src/v2/guide/components-registration.md

```diff
@@ -184,43 +184,43 @@ Just to support relatively little markup in a template:
 </BaseButton>
 ```
 
-Fortunately, if you're using Webpack (or [Vue CLI 3+](https://github.com/vuejs/vue-cli), which uses Webpack internally), you can use `require.context` to globally register only these very common base components. Here's an example of the code you might use to globally import base components in your app's entry file (e.g. `src/main.js`):
+Afortunadamente, si usas Webpack (o [Vue CLI 3+](https://github.com/vuejs/vue-cli), que usa Webpack internamente), puedes usar `require.context` para registrar globalmente sólo estos componentes base muy comunes. Aquí tienes un ejemplo del código que puedes usar para importar globalmente los componentes base en el archivo de entrada de tu aplicación (por ejemplo, `src/main.js`):
 
 ```js
 import Vue from 'vue'
 import upperFirst from 'lodash/upperFirst'
 import camelCase from 'lodash/camelCase'
 
 const requireComponent = require.context(
-  // The relative path of the components folder
+  // La ruta relativa de la carpeta de componentes
   './components',
-  // Whether or not to look in subfolders
+  // Si mirar o no en las subcarpetas
   false,
-  // The regular expression used to match base component filenames
+  // La expresión regular utilizada para hacer coincidir los nombres de los componentes básicos de los ficheros
   /Base[A-Z]\w+\.(vue|js)$/
 )
 
 requireComponent.keys().forEach(fileName => {
-  // Get component config
+  // Obtener la configuración de los componentes
   const componentConfig = requireComponent(fileName)
 
-  // Get PascalCase name of component
+  // Obtener el nombre PascalCase del componente
   const componentName = upperFirst(
     camelCase(
       // Strip the leading `./` and extension from the filename
```

Comment:
It looks like you forgot to translate this comment 😞

Reply:
Yes :( sorry for that. I will do it

Reply:
@Tintef I changed this in other commit